tools/restaked/src/main.rs

```diff
@@ -0,0 +1,119 @@
+use std::path::Path;
```

Comment:
It is kinda hacky that this tool uses a testing fixture. We should write a proper RPC client for our node and use it here. https://github.com/nearprotocol/nearcore/issues/1905